sdk/eventhub/tests.yml

```
@@ -25,4 +25,5 @@ stages:
         AZURE_CLIENT_ID: $(EVENTHUB_CLIENT_ID)
         AZURE_TENANT_ID: $(EVENTHUB_TENANT_ID)
         AZURE_CLIENT_SECRET: $(EVENTHUB_CLIENT_SECRET)
+        AZURE_SUBSCRIPTION_ID: $(python-eh-livetest-event-hub-subscription-id)
```

This needs to follow the same pattern as the others:

```suggestion
        AZURE_SUBSCRIPTION_ID: $(EVENTHUB_SUBSCRIPTION_ID)
```

---

@benbp I have updated by your comments. But it failed in UsGov, and China Cloud. Please help to resolve this problem. For more details please check [here](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1194846&view=results).
In UsGov, The value of `AZURE_SUBSCRIPTION_ID` is still problematic. Please help to confirm.
Error: `azure.core.exceptions.ResourceNotFoundError: (SubscriptionNotFound) The subscription '***' could not be found.`

In China:
Error: `azure.core.exceptions.ClientAuthenticationError: Authentication failed: AAD***T***500011: The resource principal named https://management.azure.com was not found in the tenant named MC.DevOps. This can happen if the application has not been installed by the administrator of the tenant or consented to by any user in the tenant. You might have sent your authentication request to the wrong tenant.`

---

@benbp Please help confirm the value of the parameter `AZURE_SUBSCRIPTION_ID` in UsGov and China Cloud.
Besides, about the error `azure.core.exceptions.ClientAuthenticationError` in China cloud, it's a mgmt sdk issue. After I investigate, I will create an Issue.

---

@v-xuto I will take a look. I'm also working on making these values readable in the logs so you can verify more easily.

---

@benbp Do you have any thoughts on this error `(SubscriptionNotFound) The subscription '***' could not be found` in UsGov?

---

@v-xuto I think the problem here is it's looking in the wrong tenant/authority host. Likely the environment credentials need to be called differently or updated to reference this. For example, the China error is looking for `management.azure.com` when it should be `.cn`.

I re-ran the pipeline for gov with the updated changes to print non-secret values and it is referencing the correct subscription id from the deployment, so I assume the issue is with the test code setup. https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1245762&view=results

---

@benbp By running the tests locally, I have investigated the issue clearly. The issues on `usgov` and `china`, although the error information is different, but the same problem. About the error `https://management.azure.com`, when the cloud is `usgov`, its value should be `https://management.usgovcloudapi.net`; when the cloud is `china`, its value should be `https://management.chinacloudapi.net`. This issue occurs in the eventhub [conftest.py](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/eventhub/azure-eventhub/conftest.py#L90) file when creating resource group and event hub. So we need to file issues like [textanalytics](https://github.com/Azure/azure-sdk-for-python/issues/21448) in `mgmt resource` and `mgmt eventhub` sdk.  What do you think? If so, in which repo should I file mgmt issues?